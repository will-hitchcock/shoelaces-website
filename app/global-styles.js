import { injectGlobal } from 'styled-components'

/* eslint no-unused-expressions: 0 */
export const globalStyles = injectGlobal`
  html,
  body {
    height: 100%;
    width: 100%;
  }

  body {
    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  }

  body.fontLoaded {
    font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
  }

  #app {
    color: white;
    background: linear-gradient(45deg, rgba(70,130,180,1) 0%, rgba(99,184,255,1) 100%);
    min-height: 100%;
    min-width: 100%;
  }

  p,
  label {
    font-family: Georgia, Times, 'Times New Roman', serif;
    line-height: 1.5em;
  }
`
