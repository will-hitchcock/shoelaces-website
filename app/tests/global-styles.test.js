import { globalStyles } from '../global-styles'

describe('globalStyles', () => {
  it('rendrs global styles', () => {
    expect(globalStyles).toMatchSnapshot()
  })
})
