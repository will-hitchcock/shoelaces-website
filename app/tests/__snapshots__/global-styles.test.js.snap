exports[`globalStyles rendrs global styles 1`] = `
Object {
  "#app": Object {
    "background": "white",
    "color": "#4682b4",
    "minHeight": "100%",
    "minWidth": "100%",
  },
  "body": Object {
    ".fontLoaded": Object {
      "fontFamily": "\"Roboto\", sans-serif",
    },
    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, sans-serif",
  },
  "html, body": Object {
    "height": "100%",
    "width": "100%",
  },
}
`;
